docs/static/releasenotes.asciidoc

```diff
@@ -68,6 +69,83 @@ This section summarizes the changes in the following releases:
 * <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>
 
 
+[[logstash-8-16-0]]
+=== Logstash 8.16.0 Release Notes
+
+==== Announcing the new {ls} health report API
+
+The new health report API (`GET /_health_report`) is available with {ls} `8.16.0` and later,
+with indicators that are capable of detecting degraded status of configured pipelines and
+providing actionable insights https://github.com/elastic/logstash/pull/16520[#16520], https://github.com/elastic/logstash/pull/16532[#16532].
+
+For a more detailed view please check <<logstash-health-report-api>>.
+
+[[notable-8.16.0]]
+==== Notable issues fixed
+
+* When a conditional evaluation encounter an error in the expression, the event that triggered the issue is sent to pipeline's DLQ, if enabled for the executing pipeline. https://github.com/elastic/logstash/pull/16423[#16423]
+
+* Fixed the `if` statement expression evaluation against runtime errors that could crash the pipeline, just logging the offending event and continuing with the next in the batch. https://github.com/elastic/logstash/pull/16322[#16322]
```

Comment:
You can merge these two into a single entry, talking about how Logstash will now handle conditionals that can't be evaluated, either by dropping the event or sending to the DLQ if it's enabled <link1> <link2>

Reply:
Done, could you please have a second look? Thanks!